cypress/integration/conditions/date_conditions.cy.js

```diff
@@ -0,0 +1,309 @@
+import { DIMENSION_ID_EVENT_DATE } from '../../../src/modules/dimensionConstants.js'
+import {
+    ANALYTICS_PROGRAM,
+    TEST_DIM_DATETIME,
+    TEST_DIM_DATE,
+    TEST_DIM_TIME,
+    TEST_REL_PE_LAST_12_MONTHS,
+} from '../../data/index.js'
+import {
+    openDimension,
+    selectEventProgram,
+    selectEventProgramDimensions,
+} from '../../helpers/dimensions.js'
+import { clickMenubarUpdateButton } from '../../helpers/menubar.js'
+import {
+    selectRelativePeriod,
+    getPreviousYearStr,
+    getCurrentYearStr,
+} from '../../helpers/period.js'
+import {
+    expectTableToBeVisible,
+    expectTableToMatchRows,
+} from '../../helpers/table.js'
+import { EXTENDED_TIMEOUT } from '../../support/util.js'
+
+const event = ANALYTICS_PROGRAM
+const dimensionName = TEST_DIM_DATE
+const periodLabel = event[DIMENSION_ID_EVENT_DATE]
+const stageName = 'Stage 1 - Repeatable'
+
+const setUpTable = () => {
+    selectEventProgramDimensions({ ...event, dimensions: [dimensionName] })
+
+    selectRelativePeriod({
+        label: periodLabel,
+        period: TEST_REL_PE_LAST_12_MONTHS,
+    })
+
+    clickMenubarUpdateButton()
+
+    expectTableToBeVisible()
+}
+
+const addConditions = (conditions) => {
+    cy.getBySelLike('layout-chip').contains(dimensionName).click()
+    conditions.forEach(({ conditionName, value }) => {
+        cy.getWithDataTest('{button-add-condition}').click()
```

Comment:
getBySel

Reply:
Done, I replaced all of these for the whole Cypress folder.